internal/acceptance/data_groups_test.go

```diff
@@ -0,0 +1,56 @@
+package acceptance
+
+import (
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-testing/terraform"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+const groupsDataSourceTemplate = `
+resource "databricks_group" "group1" {
+	display_name = "tf-group-{var.RANDOM}"
+}
+resource "databricks_group" "group2" {
+	display_name = "tf-groupfoo-{var.RANDOM}"
+}
+resource "databricks_group" "group3" {
+	display_name = "tf-groupbar-{var.RANDOM}"
+}
+
+#data "databricks_groups" "all" {}
+
+data "databricks_groups" "this" {
```

Comment:
You need to add `depends_on` on the group resources, otherwise they could be created after data source is executed

Reply:
Done